config/crds/resources/apiextensions.k8s.io_v1_customresourcedefinition_accesscontextmanageraccesslevels.accesscontextmanager.cnrm.cloud.google.com.yaml

```diff
@@ -43,44 +44,49 @@ spec:
     name: v1beta1
     schema:
       openAPIV3Schema:
+        description: AccessContextManagerAccessLevel is the Schema for the accesscontextmanageraccesslevels
+          API
         properties:
           apiVersion:
-            description: 'apiVersion defines the versioned schema of this representation
+            description: 'APIVersion defines the versioned schema of this representation
               of an object. Servers should convert recognized schemas to the latest
-              internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/api-conventions.md#resources'
+              internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources'
             type: string
           kind:
-            description: 'kind is a string value representing the REST resource this
+            description: 'Kind is a string value representing the REST resource this
               object represents. Servers may infer this from the endpoint the client
-              submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/api-conventions.md#types-kinds'
+              submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds'
             type: string
           metadata:
             type: object
           spec:
             properties:
               accessPolicyRef:
-                description: |-
-                  The AccessContextManagerAccessPolicy this
-                  AccessContextManagerAccessLevel lives in.
+                description: The AccessContextManagerAccessPolicy this AccessContextManagerAccessLevel
+                  lives in.
                 oneOf:
                 - not:
                     required:
                     - external
                   required:
                   - name
+                  - kind
```

Comment:
We've been bitten by this one before.  This means that users now have to specify Kind where they didn't previously.

I think the answer is that 

AccessPolicyRef v1alpha1.ResourceRef `json:"accessPolicyRef"`

instead of v1alpha1.ResourceRef should use a type AccessPolicyRef (_hopefully_ we already have that).

Bonus: we'll also get better comments on name/namespace/external